browser/ui/views/sidebar/sidebar_items_scroll_view.cc

```diff
@@ -217,7 +217,7 @@ void SidebarItemsScrollView::OnItemAdded(const sidebar::SidebarItem& item,
 
   // Calculate and set this view's bounds to determine whether this view is
   // scroll mode or not.
-  parent()->Layout();
+  DeprecatedLayoutImmediately();
```

Comment:
```suggestion
  parent()->DeprecatedLayoutImmediately();
```

Reply:
Thanks for finding that @simonhong, fixed here 106e3eb78c39cbb78f19a11d6d99937d76859eca (and workaround removed).